### PR TITLE
Implement dolweb/lobby

### PR DESCRIPTION
**Pull request by spycrab**
Prerequisite for https://github.com/dolphin-emu/dolphin/pull/7945.

### CODE REVIEW

File: dolweb/lobby/views.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                filtered_sessions.append(session)
+        else:
+            if session[key] == value:
+                filtered_sessions.append(session)
+
+
+    return filtered_sessions
+
+def _bad_method():
+    return JsonResponse({'status':'BAD_METHOD'})
```
Comment (BhaaLseN) [inline dolweb/lobby/views.py]: For consistency with `_bad_version` (and all the others) you should add some spaces here (or remove them over there)

**Comment by delroth** [comment]
The way this is written just cannot work as it is. Requests are round-robined between multiple independent workers that don't share memory. Your global "sessions" object is going to be per-worker, which means this is never going to work in production.

I also question why this has to be part of dolphin-emu/www -- it doesn't seem to share any of the codebase or the database models. It seems like it could be a standalone wsgi app written on a less heavyweight framework (Tornado, Flask, Bottle, ... pick your poison). EDIT: Or something not even written in Python -- whatever you prefer.

Finally, **before** this is live we should have a story for moderation.

**Comment by JMC47** [comment]
Having seen the Melee community, at the very least we need a disable chat button for users.  As discussed on IRC, we probably need some backend access to remove certain lobbies.  Maybe it would be best to find someone from in those communities to help with moderation.

Other than that... what else can we do?

**Comment by spycrab** [comment]
Superseeded by https://github.com/spycrab/netplay-index

**Pull request closed without merging**